db/Pos/DB/Pure.hs

```diff
@@ -142,3 +147,71 @@ dbWriteBatchPureDefault (tagToLens -> l) batchOps =
 
 atomicModifyIORefPure :: (MonadIO m) => (a -> a) -> IORef a -> m ()
 atomicModifyIORefPure foo = flip atomicModifyIORef $ \a -> (foo a, ())
+
+data ByteStringDiff = ByteStringDiff
```

Comment:
I know it's almost obvious what it does but the policy is to document everything exported if possible.